i18n/en-US.properties

```diff
@@ -746,6 +746,18 @@ boxui.datePicker.iconAlertText = Invalid Date
 boxui.draftjs.mentionSelector.startMention = Mention someone to notify them
 # Icon showing a sad Box cloud
 boxui.errorMask.iconSadCloudText = Sad Box Cloud
+# Text to show for root folder is external
+boxui.features.CellRenderers.allFiles = All Files
+# The user is a anonymous user
+boxui.features.CellRenderers.anonymousUser = Anonymous User
+# Text to show when a file is external
+boxui.features.CellRenderers.externalFile = External File
+# Text to show when a folder is external
+boxui.features.CellRenderers.externalFolder = External Folder
+# Text to show on "modified by" table cell. Note that "lastModified" will contain additional localized text. Example: 2 days ago by John Smith
+boxui.features.CellRenderers.lastModifiedBy = {lastModified} by {user}
```

Comment:
might want to check with the translation team on this - I'd guess that injecting a pretranslated substring doesn't work well in several languages